client/state/reader/start/reducer.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import keyBy from 'lodash/keyBy';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_START_RECOMMENDATIONS_RECEIVE,
+	READER_START_RECOMMENDATIONS_REQUEST,
+	READER_START_RECOMMENDATIONS_REQUEST_SUCCESS,
+	READER_START_RECOMMENDATIONS_REQUEST_FAILURE,
+	SERIALIZE,
+	DESERIALIZE,
+} from 'state/action-types';
+import { itemsSchema } from './schema';
+import { isValidStateWithSchema } from 'state/utils';
+
+/**
+ * Tracks all known list objects, indexed by list ID.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function items( state = {}, action ) {
+	switch ( action.type ) {
+		case READER_START_RECOMMENDATIONS_RECEIVE:
+			return Object.assign( {}, state, keyBy( action.recommendations, 'site_ID' ) );
```

Comment:
probably want a new tree sections for reader sites and posts? do we start that move now?